nuxeo-opencmis-impl/src/main/resources/OSGI-INF/properties-contrib.xml

```diff
@@ -7,6 +7,7 @@
       Note that setting this to true gives a behavior that is contrary to the CMIS specification.
     </documentation>
```

Comment:
Please update the documentation to explain your second property, and mention in the existing text that it's for the `org.nuxeo.cmis.errorOnCancelCheckOutOfDraftVersion` property.